aikau/src/main/resources/alfresco/lists/Paginator.js

```diff
@@ -105,6 +115,18 @@ define(["dojo/_base/declare",
        * @default
        */
       currentPage: null,
+
+      /**
+       * The style of current page number. The default is "NUMBER" which indicates that just the
+       * page number should be displayed. The only alternative in the current version is
+       * "X_OF_Y" which will render the label in the form "1 of 4" (as in page 1 of 4).
+       * 
+       * @instance
+       * @type {string}
+       * @default
+       * @since 1.0.64
+       */
+      currentPageLabelStyle: "NUMBER",
```

Comment:
Is it worth making this a format mask instead, e.g. `"{pageNum} of {numPages}"` or `"Numero {pageNum}"`? Dunno how much extra effort is involved and whether there would be any problems with i18n. Perhaps it's a supplementary feature?